src/main/resources/assets/nutrition/configs/nutrients/protein.json

```diff
@@ -748,6 +749,14 @@
 			"tconstruct:edible:22",
 			"tconstruct:edible:23",
 			"tconstruct:edible:33",
+			"thaumcraft:brain",
+			"thaumcraft:chunk",
```

Comment:
Can you mark `thaumcraft:chunk` as `thaumcraft:chunk:0`, just to be more explicit?  Leaving the `:0` off is fine in most cases, but I prefer specifying the metadata when it's relevant.